scripts/inject.js

```diff
@@ -36,6 +37,18 @@ api.ApiPromise.create({provider: wsProvider}).then(function (instance) {
 
     console.log("Connected");
     replacements.forEach(async function (replacement) {
+        if (replacement.computed) {
```

Comment:
How does one test the injected code on the learn-proxies page?

Reply:
The way I tested it was by generating the static site using `yarn polkadot:build` then using `yarn polkadot:inject` to make sure it picked up the templates.